src/suncontrol/i/suncontrol_i.c

```diff
@@ -0,0 +1,192 @@
+/* -----------------------------------------------------------------
+ * Programmer(s): Daniel R. Reynolds @ SMU
+ * -----------------------------------------------------------------
+ * SUNDIALS Copyright Start
+ * Copyright (c) 2002-2023, Lawrence Livermore National Security
+ * and Southern Methodist University.
+ * All rights reserved.
+ *
+ * See the top-level LICENSE and NOTICE files for details.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ * SUNDIALS Copyright End
+ * -----------------------------------------------------------------
+ * This is the implementation file for the SUNControl_I module.
+ * -----------------------------------------------------------------*/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <suncontrol/suncontrol_i.h>
+#include <sundials/sundials_math.h>
+
+
+/* ---------------
+ * Macro accessors
+ * --------------- */
+
+#define SC_I_CONTENT(C)     ( (SUNControlContent_I)(C->content) )
+#define SC_I_K1(C)          ( SC_I_CONTENT(C)->k1 )
+#define SC_I_BIAS(C)        ( SC_I_CONTENT(C)->bias )
+#define SC_I_P(C)           ( SC_I_CONTENT(C)->p )
+#define SC_I_PQ(C)          ( SC_I_CONTENT(C)->pq )
+
+/* ------------------
+ * Default parameters
+ * ------------------ */
+
+#define DEFAULT_K1     RCONST(1.0)
+#define DEFAULT_BIAS   RCONST(1.5)
```

Comment:
I suggest making this 1 along with a default safety factor of 0.9 in the heuristics.